tests/kernel_bundle/sycl_join_kernel_bundles_with_diff_ctx.cpp

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
```

Comment:
Comments inside comments are even more confusing.
```suggestion
  
   SYCL 2020 Conformance Test Suite
  
```

Reply:
@keryell, this header is recommended in our [coding guidelines](https://github.com/KhronosGroup/SYCL-CTS/blob/SYCL-2020/docs/README.adoc#copyright-header) and it's the same for all source code files in this project. I suggest we merge this PR "as is" and open a separate issue for changing the guidelines and updating the source code base. Does it sound okay to you?

Reply:
I completely forgot about this coding guide. I should look at it. :-)
Yes, you can merge this.